src/cm.css

```diff
@@ -131,7 +131,7 @@ a:focus {
   color: var(--text-color);
   margin: 0 0 18px 0;
   text-align: center;
-  font-family: "Rethink Sans", sans-serif;
+  font-family: "Rethink Sans", "Neue Haas Unica Bold", sans-serif;
```

Comment:
Please check the website cookie banner:
<img width="716" alt="image" src="https://github.com/user-attachments/assets/6604d2ae-ccec-4b38-9a05-7434e57415cc">

It has font-weight 700
<img width="289" alt="image" src="https://github.com/user-attachments/assets/8aefa78a-50ce-4d74-8538-04c644834e37">

---

Look at how it looks on the webapp with font-weight 700 (too bold)
<img width="728" alt="image" src="https://github.com/user-attachments/assets/98f1265a-5c5f-4886-84b2-ce2200ce0feb">


When changing from 700 to 500, IMO it looks better:
<img width="727" alt="image" src="https://github.com/user-attachments/assets/de654b3d-ab29-4c6f-9568-b1babe8c92df">

--- 

Given that said, I would recommend the following
1. using the this font config: `Rethink Sans / 700` and `neue-haas-unica / 500`
2. changing from `"Neue Haas Unica Bold"` to `neue-haas-unica` (this is the font definition in the website)